app/modules/sightings/models.py

```diff
@@ -1420,8 +1455,20 @@ def identified(self, job_id, data):
                 f'Sighting:{self.guid}, Annot:{annot_guid}, algorithm:{algorithm}'
             )
 
+            annotation = Annotation.query.get(annot_guid)
+            if not annotation:
+                raise HoustonException(
+                    log, f'annotation {annot_guid} for {job_id_str} not found'
+                )
+
+            if annotation.progress_identification is not None:
```

Comment:
just curious why you use `... is not None` for these, but above just `if annotation.progress_identification:` ?

Reply:
Good catch, will fix